draft-twi-reference-architecture.md

```diff
@@ -57,28 +57,77 @@ This document uses terms and concepts defined by the WIMSE and RATS architecture
 see {{Section 4 of -rats-arch}} & {{-WIMSE}}.
 
 
-## Glossary
+# Glossary
 {: #sec-glossary }
 
 This document uses the following terms:
 
+## Workload Identity
+
+## Workload Credential
+
+
 # TWI Use Cases
 Following are some of the use cases demanding Trustworthy Workload Identity.
 
 1. A ML Model Owner (acting as Relying Party) wants to deploy its Workload with a unique identity in a privacy
 preserving manner. It needs to ensure that the deployed workload is operating in an environment with certain
 security guarantees, such as isolated from cloud hosting environment and can be periodically attested.
 
-
 2. A Workload operating as a deployed lambda running in an isolated environment.
 
 # Core Requirements
+The TWI Core Requirements can be located here. (TO DO Add link here from Mark)
+
+# Alignment or Synergy with WIMSE Architecture
+WIMSE defines an architecture for managing workload identity in multi-system environments.
+
+1. The WIMSE Architecture, explains the core concept of Workload Identity in-line with the concept of identity in a TWI world.
+
+2. WIMSE model has a CA/Credential issuer that is responsible with provisioning identity credentials to the workload.
```

Comment:
issuing, not provisioning. Provisioning is the process of obtaining a credential from an issuer, which is done by the workload or the workload owner.